tests/hpc/slurm_master.pm

```diff
@@ -425,15 +437,22 @@ sub run ($self) {
     my $nodes = get_required_var('CLUSTER_NODES');
     my $slurm_conf = get_required_var('SLURM_CONF');
     my $version = get_required_var('VERSION');
-
+    if ($slurm_pkg =~ /slurm/) {
```

Comment:
This complicate structure is not needed at all,  just do:
```
rpm -q --queryformat='%{VERSION}' $slurm_pkg
```

Reply:
Done, ready for review